marlowe-playground-client/src/Marlowe/Linter.purs

```diff
@@ -127,6 +131,8 @@ instance showWarningDetail :: Show WarningDetail where
   show NegativePayment = "The contract can make a non-positive payment"
   show NegativeDeposit = "The contract can make a non-positive deposit"
   show TimeoutNotIncreasing = "Timeouts should always increase in value"
+  show (InvalidAddress addr) = show addr <>
+    " is not a valid Shelley payment key address"
```

Comment:
If someone enters a Byron address, should the linter message state that Byron addresses are not allowed in Plutus contracts? or is this message sufficient in that case, too.

Reply:
That would certainly be more user friendly. But then we would have to implement Byron address validation too, and I am not sure it is worth it. The error message does state it needs to be Shelley, and people with Byron addresses probably have already been burned by other tools